build/Packages.props

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <MSBuildPackageVersion>16.6.0</MSBuildPackageVersion>
     <NuGetPackageVersion>5.2.0</NuGetPackageVersion>
-    <RoslynPackageVersion>3.8.0-1.20363.1</RoslynPackageVersion>
+    <RoslynPackageVersion>3.8.0-2.20413.3</RoslynPackageVersion>
```

Comment:
from where do you take the exact version? from https://github.com/dotnet/versions/tree/master/build-info/dotnet/roslyn? I normally go there but I wonder if there is a better place

Reply:
In this case I am taking it from the sdk, https://github.com/dotnet/sdk/blob/release/5.0.1xx-preview8/eng/Versions.props#L115, but development builds are published to AzDO as well https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-tools&package=Microsoft.Net.Compilers.Toolset&version=3.8.0-3.20418.5&protocolType=NuGet&view=versions

Reply:
I see thanks

Reply:
Also, I did not know about dotnet/versions. That is super useful. =)